test/e2e/network/netpol/k8s_util.go

```diff
@@ -33,64 +33,68 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
-// Scenario provides a convenience interface to kube functionality that we leverage for polling NetworkPolicy connections.
```

Comment:
may as well  add periods to the ends of the sentences  while were at it since this is a cleanup pr

Reply:
👍